keras_nlp/layers/fnet_encoder.py

```diff
@@ -21,12 +21,11 @@
 class FNetEncoder(keras.layers.Layer):
     """FNet encoder.
 
-    This class follows the architecture of FNet encoder layer in paper
-    "FNet: Mixing Tokens with Fourier Transforms"
-    (https://arxiv.org/abs/2105.03824). Users can instantiate multiple instances
-    of this class to stack up the encoder.
+    This class follows the architecture of FNet encoder layer in the
+    [FNet paper](https://arxiv.org/abs/2105.03824). Users can instantiate
+    multiple instances of this class to stack up the encoder.
 
-    Note on padding: In the official FNet code, padding tokens are added to the
+    Note on masking: In the official FNet code, padding tokens are added to the
     the input. However, the padding masks are deleted, i.e., mixing of
```

Comment:
Rereading this - I feel the sentence after i.e., makes it more confusing. What is "mixing of all tokens"?

Reply:
I think a reference to the paper basically. You can view attention as a mechanism for "mixing tokens" together and a fourier as an alternative mixing approach.

Anyway, I just wanted to clarify that this is relevant to the usual masking behavior of an encoding block.